macros/first_value.sql

```diff
@@ -0,0 +1,19 @@
+{% macro first_value(first_value_field, partition_field, order_by_field, order="asc") -%}
+
+{{ adapter.dispatch('first_value', packages = fivetran_utils._get_utils_namespaces()) (first_value_field, partition_field, order_by_field, order="asc") }}
```

Comment:
should `order="asc"` be included in the dispatch call?

Reply:
Nope, removed!